docs/UserGuide.md

```diff
@@ -89,18 +89,18 @@ started!
 
 1. Ensure you have Java 11 or above installed in your Computer.
 
-   __How to check?__
+   ***How to check?***
     1. Open up the Terminal on your computer.
         1. For Windows users, locate it by typing in “Terminal” in the search bar.
         2. For Mac users, locate it by typing in “Terminal” in Spotlight.
-    2. Type in java -version and press Enter.
+    2. Type in `java -version` and press Enter.
     3. If you have Java installed, you should see your Java version as shown underlined in the picture below.
        <img src='images/javaversion.png' width='500'>
     4. If you do not have Java or your Java version is below 11, install Java
        11 [here](https://www.oracle.com/java/technologies/downloads/#java11)
 
 
-2. Go to your Desktop and create a folder named “A&E”.
+2. Go to your Desktop and create a folder named “AAndE”.
```

Comment:
Thank you for standardizing the format of the name of the folder and all throughout!